toml-benchmarks.cabal

```diff
@@ -51,21 +51,31 @@ common common-options
                        TypeApplications
                        ViewPatterns
 
-library
-  import:              common-options
-  hs-source-dirs:      src
-  exposed-modules:     TomlBenchmarks
-
 
-benchmark toml-benchmarks-benchmark
```

Comment:
@chshersh I had to change this to `library` due to some weird cabal error telling me it cannot run benchmarks, did I do it correctly or should I have done something else? 😅

Reply:
@kutyel It should be `benchmark decoding`. We don't need a library in this project, since it's only for benchmark purposes. It's strange that cabal has problems :thinking: I just tried to run `cabal bench` command and it works without problems.